docs/Manual/Programs/Starter/README.md

```diff
@@ -1,7 +1,20 @@
 # ArangoDB Starter
 
-This chapter documents the _ArangoDB Starter_.
+The _ArangoDB Starter_ is a tool that can help you deploy ArangoDB in an easy way
+(either in _Single Instance_, _Active Failover_ or _Cluster_ mode).
 
-The _ArangoDB Starter_ is a tool that can help you deploy ArangoDB in an easy way (either in single-instance, active/passive or Cluster mode).
+The binary is called `arangodb`, not to be confused with the server binary `arangod`.
 
-For a Tutorial, please refer to [this](../../Tutorials/Starter/README.md) section.
+Also see:
```

Comment:
This is a bit strange use of the English language. I think "See also:" is more common,